Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -1,17 +1,19 @@
 using System.Linq;
-using System.Net.Http;
 using System.Text;
-using System.Text.Json;
-using System.Text.Json.Nodes;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Content.Server.Administration.Managers;
 using Content.Server.Afk;
 using Content.Server.Discord;
 using Content.Server.GameTicking;
+using Content.Server.Maps;
+using Content.Shared.Access.Systems;
 using Content.Shared.Administration;
 using Content.Shared.CCVar;
 using Content.Shared.Mind;
+using Discord;
+using Discord.Rest;
+using Discord.WebSocket;
```

Comment:
There should be no references to Discord.Net in this file.

In fact, ideally there wouldn't even be references to *Discord* in this file. There should be a separate manager in charge of bridging bwoinks to Discord. You'd have:
* `BwoinkSystem` (this shouldn't be a system but that's not your fault)
* `DiscordLink`
* `DiscordBwoinkLink`
* `DiscordChatLink`